go-app-ussd_popi_faq.js

```diff
@@ -63,32 +61,7 @@ go.app = function() {
             ;
         };
 
-        self.add = function(name, creator) {
-            self.states.add(name, function(name, opts) {
-                if (!interrupt || !utils.timed_out(self.im))
-                    return creator(name, opts);
-
-                interrupt = false;
-                var timeout_opts = opts || {};
-                timeout_opts.name = name;
-                return self.states.create('state_timed_out', timeout_opts);
-            });
-        };
-
-        self.states.add('state_timed_out', function(name, creator_opts) {
-            return new ChoiceState(name, {
-                question: $('Welcome back. Please select an option:'),
-                choices: [
-                    new Choice(creator_opts.name, $('Continue viewing FAQ')),
-                    new Choice('state_start', $('Main menu'))
-                ],
-                next: function(choice) {
-                    return choice.value;
-                }
-            });
-        });
```

Comment:
So the reason we have these timeout states is although we can have 182 (I think?) chars per ussd message, some providers have additional information in the first message, so the first message of a session has a lower limit (I think around 140 chars).

I just want to make sure that we've checked that none of the screens or their translations are over the 140 char limit.

The other option (which I think might be better for the FAQ, since it's not a long registration flow), is to redirect them to the main menu every time they redial after timeout.